Taskfile.yml

```
@@ -0,0 +1,301 @@
+          
+          # Clone repository if not exists
+          if [ ! -d \"/root/kosnip\" ]; then
+            echo \"📥 Cloning repository...\"
+            nix-shell -p git --run \"cd /root/ && git clone https://github.com/ZeroDeth/kosnip.git\"
+          else
+            echo \"🔄 Updating existing repository...\"
+            cd /root/kosnip && git pull
+          fi
+          
+          cd /root/kosnip/hosts/nixos/{{.HOST}}/
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)